kong/router/atc.lua

```diff
@@ -344,6 +344,16 @@ local function set_upstream_uri(req_uri, match_t)
 end
 
 
+-- captures has the form { [0] = full_path, [1] = capture1, [2] = capture2, ... }
```

Comment:
@StarlightIbuki I think the `captures` might have string keys. Am I right @chronolaw ?

Reply:
Of course, that is named capture.